tests/js/plugin/enhancedanchoring-test.coffee

```diff
@@ -688,5 +691,6 @@ describe 'Annotator.Plugin.EnhancedAnchoring', ->
             renderPage am.document, 9
             renderPage am.document, 10
 
-        anchor.scrollIntoView().then ->
-          assert.called anchor.highlight[10].scrollIntoView
+        anchor.scrollToView()
+        watchForScroll().then ->
```

Comment:
> Same here. I'm not following the flow.

See my explanation above. If you think the test code could be made more self-evident, I am open to suggestions.